includes/RestApiClient/Model/Address.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dation\Woocommerce\RestApiClient\Model;
```

Comment:
Models in the RestApiClient namespace feels a bit weird, I realise they are representations of the models we provide in our API but they're still just DTO classes usable in the entire codebase.

Reply:
Moved from includes/RestApiClient/Model to includes/Model